pkg/maintainer/btcdiff/config.go

```diff
@@ -0,0 +1,29 @@
+package btcdiff
+
+import "time"
+
+// Config contains maintainer configuration.
+type Config struct {
+	// Enabled indicates whether the Bitcoin difficulty maintainer
+	// should be started.
+	Enabled bool
+
+	// DisableProxy indicates whether the Bitcoin difficulty
+	// maintainer proxy should be disabled. By default, the Bitcoin difficulty
+	// maintainer submits the epoch headers to the relay via the proxy. If this
```

Comment:
Let's add some extra context:

"... via the proxy to be reimbursed for the transaction."

Reply:
4ee460edecc90e93e3047c04dfd5a9f26f84db3c